### PR TITLE
[TechDraw] Improve readability of App scripts, part 4

### PR DESCRIPTION
**Pull request by benj5378**
### [TechDraw] Improve readability of App scripts, part 4
- [[TechDraw] Improve readability of DrawViewSection.cpp](https://github.com/FreeCAD/FreeCAD/commit/3e650514753660880cf6b57736d618150e21016e) 
- [[TechDraw] Improve readability of DrawViewSpreadsheet.cpp](https://github.com/FreeCAD/FreeCAD/commit/d93f5e33067311598a03425d7487d12bf58653fe)
- [[TechDraw] Improve readability of DrawWeldSymbol.cpp](https://github.com/FreeCAD/FreeCAD/commit/e47de75e7d6f36c960de77de3a02a171e0f9f7db)
- [[TechDraw] Improve readability of DrawViewSymbol.cpp](https://github.com/FreeCAD/FreeCAD/commit/15a891b339a7a94d59ebf52880f3e77a98f5032a)

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [x]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [x]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [x]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [x]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [x]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the issue ID number from our [Issues database](https://github.com/FreeCAD/FreeCAD/issues) in case a particular commit solves or is related to an existing issue. Ex: `Draft: fix typos - fixes #4805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

### CODE REVIEW

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7209">pipeline status</a> for feature branch [PR_7209](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7209). Pipeline [589844568](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/589844568) was triggered at [15a891b](https://github.com/FreeCAD/FreeCAD/pull/7209/commits/15a891b339a7a94d59ebf52880f3e77a98f5032a). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/TechDraw/App/DrawViewSpreadsheet.cpp [hunk 1]
```diff
@@ -177,25 +174,26 @@ std::string DrawViewSpreadsheet::getSheetImage(void)
...
+            Base::Console().Error("%s - start cell (%s) invalid row\n",
+                                    getNameInDocument(), rowPart.c_str());
+            return std::string();
         }
     }
 
     if (boost::regex_search(scellend, what, re)) {
         if (what.size() < 3) {
             Base::Console().Error("%s - end cell (%s) is invalid\n",getNameInDocument(),CellEnd.getValue());
+            return std::string(); // shouldn't the function be returned here too?
```
Comment (benj5378) [inline src/Mod/TechDraw/App/DrawViewSpreadsheet.cpp:196]: @WandererFan

**Comment by benj5378** [comment]
@WandererFan Ready for review

**Comment by WandererFan** [comment]
DrawViewSymbol has been changed quite a bit. See #7266.
DrawViewSection is going to change for multiThreading.  see my branch https://github.com/WandererFan/FreeCAD/tree/hlrThreadrc1

Spreadsheet and WeldSymbol look alright.

**Comment by benj5378** [comment]
>

**Comment by benj5378** [comment]
> DrawViewSection is going to change for multiThreading. see my branch https://github.com/WandererFan/FreeCAD/tree/hlrThreadrc1
So what are you suggesting, that we leave DrawViewSection out for now? @WandererFan

**Comment by WandererFan** [comment]
I would leave it out.

**Comment by benj5378** [comment]
> I would leave it out.

Removed @WandererFan

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7209">pipeline status</a> for feature branch [PR_7209](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7209). Pipeline [600422116](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/600422116) was triggered at [1b58fa9](https://github.com/FreeCAD/FreeCAD/pull/7209/commits/1b58fa9a386f7e6ee78a974919f928321283f2bf). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Pull request merged**